crates/settings_ui/src/page_data.rs

```diff
@@ -1432,7 +1432,7 @@ pub(crate) fn settings_data(cx: &App) -> Vec<SettingsPage> {
                     }),
                     SettingsPageItem::SettingItem(SettingItem {
                         title: "Relative Line Numbers",
-                        description: "Whether the line numbers in the editor's gutter are relative or not",
+                        description: "Whether to use relative line numbers, or relative wrapped lines, in the editor's gutter",
```

Comment:
Maybe we can make this a bit more descriptive? Something like
```suggestion
                        description: "Controls line number display in the editor's gutter. "disabled" shows absolute line numbers, "enabled" shows relative line numbers for each absolute line, and "wrapped" shows relative line numbers for every line, absolute or wrapped.",
```

Though that might be more verbose than you want it? Up to you 💁🏼